optimum/intel/neural_compressor/quantization.py

```diff
@@ -266,45 +282,24 @@ def quantize(
                 logger.warning("ONNX export is no supported for model with quantized embeddings")
                 save_onnx_model = False
 
-        else:
-            # Disable ONNX export for dynamically quantized model as deprecated in neural-compressor>=2.2.0
-            if save_onnx_model:
-                logger.warning(
-                    "ONNX export for dynamic quantized model is no longer supported by neural-compressor>=2.2.0. "
-                    "To apply dynamic quantization on an ONNX model, you can use optimum.onnxruntime.ORTQuantizer"
-                )
-                save_onnx_model = False
-
-        if (
-            isinstance(quantization_config, PostTrainingQuantConfig)
-            and quantization_config.backend == "ipex"
-            and is_ipex_version("<", IPEX_MINIMUM_VERSION)
-            and "generation" in self.task
-        ):
-            raise ImportError(
-                f"Found an incompatible version of intel-extension-for-pytorch. Found version {_ipex_version}, "
-                f"but only version {IPEX_MINIMUM_VERSION} or higher is supported."
-            )
-
         if not isinstance(quantization_config, PostTrainingQuantConfig):
             if use_cpu:
                 # will remove after intel-extension-for-transformers 1.3.3 release.
                 quantization_config.device = "cpu"
-                quantization_config.post_init()
+                quantization_config.post_init_cpu()
             elif use_xpu:
                 # will remove after intel-extension-for-transformers 1.3.3 release.
                 quantization_config.device = "xpu"
                 quantization_config.post_init_xpu()
+
             self._quantized_model = convert_to_quantized_model(
                 self._original_model, quantization_config, device=quantization_config.device
             )
-            # will remove after intel-extension-for-transformers 1.3.3 release.
-            if hasattr(quantization_config, "calib_dataloader"):
-                quantization_config.calib_dataloader = None
+
             self._quantized_model.quantization_config = quantization_config
```

Comment:
```suggestion
            self._quantized_model.quantization_config = quantization_config
            self._quantized_model.config.quantization_config = quantization_config
```

Reply:
In the ITREX loading function, 'quantization_config' in 'config' distinguishes between quantized models and those requiring quantization. It is aligned with other quantization methods, like AutoGPTQ, AutoAWQ, and Bitsandbyte.